source/_integrations/august.markdown

```diff
@@ -43,6 +43,7 @@ The `august` integration allows you to integrate your [August](https://august.co
 | Yale Assure Lock 2 | yes |
 | Yale Conexis L1 | yes |
 | Yale Conexis L2 | yes |
+| Yale Doorman L3 | yes |
```

Comment:
Do you think "L3S" should be mentioned?

Reply:
Historically, we haven't mentioned all the sub models since if the base model works usually all the sub models do... and there are a lot of them for each country in some cases

Reply:
Make sense.